cimi-deployment/README.md

```diff
@@ -0,0 +1,77 @@
+# Deploying CIMI with Docker
+
+This document describes how you can run the CIMI server using Docker
+containers.
+
+## Prerequisites
+
+Ensure that [Docker](https://www.docker.com/) and [Docker
+Compose](https://docs.docker.com/compose/) installed.
+
+Make sure that you've allocated enough memory to Docker; 4-6 GB of
```

Comment:
this is just if you want to use ES, and want to give it 2GB of RAM. It is a very specific configuration. Should be more like a recommendation in case you want to use ES